assume/strategies/flexable.py

```diff
@@ -19,6 +20,7 @@ def calculate_bids(
         bid_quantity_flex, bid_price_flex = 0, 0
 
         if operational_window is not None:
+            self.current_time = operational_window["window"]["start"]
```

Comment:
This is wrong - the start of the bidding window is not the current time

Reply:
yes, you are correct. This is still partially work in progress, this will change to better naming.